.circleci/config.yml

```diff
@@ -706,30 +706,30 @@ workflows:
           requires:
             - build
       - create-sandboxes:
-          parallelism: 32
+          parallelism: 33
```

Comment:
I don't get this, as I don't see an extra template.

Reply:
I removed `inDevelopment: true` for one of the sandboxes in `code/lib/cli/src/sandbox-templates.ts`